doc/faq.rst

```diff
@@ -10,6 +10,23 @@ Altair Frequently Asked Questions
 
 .. _altair-faq-no-display:
 
+Does Altair work in more modern frontends such as JupyterLab or nteract
+-----------------------------------------------------------------------
+
+Yes! In fact, JupyterLab and nteract ship with builtin rendering for Vega
+and VegaLite. This functionality is still being finalized, but this version
```

Comment:
Since this will eventually be on the website, can we change "this version" to the specific version number to which it applies?